engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -142,7 +142,7 @@ <h4 translate>Content Host Properties</h4>
 
       <div class="detail">
         <span class="info-label" translate>RAM </span>
```

Comment:
@kgaikwad thanks for reviewing. I have made the necessary changes. 

Let me know any further changes are required.